app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb

```diff
@@ -6,6 +6,12 @@ oses:
 - CentOS
 - Fedora
 - RedHat
+test_on:
```

Comment:
Can we phrase this in a way that's also tells the user something? Perhaps `networking`?

Reply:
Well stubs can change anything, not just networking. Better ideas?

Reply:
Maybe `supported_on`? and make it `ipv4dhcp`, `ipv4and6dhcp`? :thinking: